From e40e628436818552a2d1ef18a1c04b644fe56e3f Mon Sep 17 00:00:00 2001
From: Hong-Mei Li <a21834@motorola.com>
Date: Fri, 25 Sep 2015 23:19:22 -0700
Subject: [PATCH 219/959] IKSWM-4458 motoconfig: Enable support for Large
 Receive Offload

In computer networking, large receive offload is a technique for increasing
inbound throughput of high-bandwidth network connections by reducing CPU overhead.
It works by aggregating multiple incoming packets from a single stream into
a larger buffer before they are passed higher up the networking stack,
thus reducing the number of packets that have to be processed.
In Linux, it is generally used in conjunction with the New API (NAPI) to also
reduce the number of interrupts.
According to benchmarks, even implementing this technique entirely in software
can increase network performance significantly.

Change-Id: I81b5a72161aab3d6ace15e876af483e5f895daf6
Signed-off-by: Hong-Mei Li <a21834@motorola.com>
---
 arch/arm/configs/ext_config/moto-msm8916.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/ext_config/moto-msm8916.config b/arch/arm/configs/ext_config/moto-msm8916.config
index 54976981d77..32c8129e949 100644
--- a/arch/arm/configs/ext_config/moto-msm8916.config
+++ b/arch/arm/configs/ext_config/moto-msm8916.config
@@ -41,3 +41,4 @@ CONFIG_NETFILTER_XT_MATCH_TCPMSS=y
 CONFIG_IP_MULTICAST=y
 CONFIG_IP_MROUTE=y
 CONFIG_IP_PIMSM_V2=y
+CONFIG_INET_LRO=y
-- 
2.11.0

